server/src/main/resources/application.yml

```diff
@@ -4,7 +4,7 @@ lighter:
     spark-history-server-url: http://localhost/spark-history/
     frontend-path: file:${FRONTEND_PATH:../frontend/build}
     py-gateway-port: 25333
-    url: http://lighter.spark:8080
+    url: http://localhost:8080
```

Comment:
Lets not change it on the main application file, to run the lighter locally, there is a `application-local.yaml`, to activate it you can set env var `MICRONAUT_ENVIRONMENTS=local` or set run property `-Dmicronaut.environments=local`

Reply:
sorry my bad

Reply:
done